alfresco-content-services/values.yaml

```diff
@@ -950,7 +968,7 @@ global:
     auth: secret
     # -- Shared secret to authenticate repo/solr traffic.
     # Strong enough secret can be generated with `openssl rand 20 -base64`
-    sharedsecret: null
+    sharedsecret: sharedsecret
```

Comment:
if im not mistaken "sharedsecret" is simply the secret vaule in plain text. ultimately we should not be committing a secret to the repo. would it be possible to generate a kubernetes secret through a template?

alternatively, as we are still in poc stages, we could leave this as is and add a ticket to our backlog to improve this before we start dev testing/migration work

Reply:
Yes Pete, agree..I think we should add a ticket to our backlog to do this. For this PoC context, if we have share and cs both up and running, we'll start cleaning up in our next sprint. Is that okay?